File: src/main/java/io/github/oleksivio/telegram/bot/api/model/method/location/StopOwnMessageLiveLocation.java
package io.github.oleksivio.telegram.bot.api.model.method.location;

import io.github.oleksivio.telegram.bot.api.model.objects.std.Message;
import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker;
import io.github.oleksivio.telegram.bot.core.model.method.location.StopMessageLiveLocation;
import io.github.oleksivio.telegram.bot.core.model.responses.MessageResponse;
import io.github.oleksivio.telegram.bot.core.model.responses.CommonResponse;

public class StopOwnMessageLiveLocation extends StopMessageLiveLocation<Message> {
    public StopOwnMessageLiveLocation(ActionNetworker actionNetworker) {
        super(actionNetworker);
    }

    @Override
    protected Class<? extends CommonResponse<Message>> getResultWrapperClass() {
        return MessageResponse.class;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/method/location/StopOwnMessageLiveLocation.kt
package io.github.oleksivio.telegram.bot.api.model.method.location

import io.github.oleksivio.telegram.bot.api.model.objects.std.Message
import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker
import io.github.oleksivio.telegram.bot.core.model.MessageResponse
import io.github.oleksivio.telegram.bot.core.model.method.location.StopMessageLiveLocation

class StopOwnMessageLiveLocation(actionNetworker: ActionNetworker) : StopMessageLiveLocation<Message>(actionNetworker) {
    override val resultWrapperClass = MessageResponse::class
}
